android/build.gradle

```diff
@@ -37,22 +44,34 @@ def useDedicatedExecutor = rootProject.hasProperty('AsyncStorage_dedicatedExecut
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
-    buildToolsVersion getExtOrDefault('buildToolsVersion')
-
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
     defaultConfig {
-        minSdkVersion getExtOrIntegerDefault('minSdkVersion')
-        targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
-
+        minSdkVersion safeExtGet('minSdkVersion', 19)
```

Comment:
I'm happy keeping 19 here, but as @dulmandakh said, it would be better to comply with RN's 16.

Reply:
Sure, I agree it would be best to comply with 16 from react-native. Is it okay to make this change in a separate PR? Lowering the minimum SDK _could_ require other code changes. Since this PR is not about change the SDK level, it would keep these changes more focussed (and easier to review).

Reply:
You're right, let's keep 19 for now